extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/memo.stack/lib/viewport_placement_utils.py

```diff
@@ -0,0 +1,179 @@
+from pyrevit import revit, script, forms, DB
```

Comment:
I've moved all methods which are used both in Copy and Paste scripts to separated script in `lib`